itests/org.openhab.core.model.rule.tests/itest.bndrun

```diff
@@ -126,10 +124,11 @@ Fragment-Host: org.openhab.core.model.rule.runtime
 	org.objectweb.asm.tree;version='[9.6.0,9.6.1)',\
 	org.objectweb.asm.tree.analysis;version='[9.6.0,9.6.1)',\
 	org.objectweb.asm.util;version='[9.6.0,9.6.1)',\
-	org.openhab.core.model.persistence.runtime;version='[4.2.0,4.2.1)',\
```

Comment:
This removal seems unrelated. Any idea why it is there and if it is ok to be removed?

Reply:
I just let the resolver resolve the bundles and it came up with these bundles. The tests still run fine without it. There is probably some implementation detail that makes it resolve different bundles depending on the local environment. Maybe it's due to the OS, Java version or the order of files on disk. To workaround it we could blacklist those bundles that continuously get added/removed by the resolver.

Reply:
Ok, thanks!